frontstage/controllers/collection_instrument_controller.py

```diff
@@ -86,3 +86,14 @@ def upload_collection_instrument(upload_file, case_id, party_id):
         raise ApiError(logger, response)
 
     logger.info('Successfully uploaded collection instrument', case_id=case_id, party_id=party_id)
+
+
+def check_collection_instrument_size(upload_file, party_id, case_id):
+    logger.info('Checking the file size', party_id=party_id, case_id=case_id)
+    upload_file_readable = upload_file.read()
+    upload_file_size = len(upload_file_readable)
+    print('file size is: ', upload_file_size)
+
+    if upload_file_size > 0:
+        logger.info('File size is correct', party_id=party_id, case_id=case_id)
+        return True
```

Comment:
So what happens if it is too big?  You don't `return False` and you don't raise an exception.  Currently if the file is 0 bytes then it implicitly does a `return None`.
Having a function with 2 return types (in this case bool or None) is bad and will bite you down the line.
I think having this return False is good enough (especially if you do the function rename I mention earlier)